### PR TITLE
fix: add support for spread operators and object methods

### PR DESCRIPTION
**Pull request by dword-design**

### CODE REVIEW

File: tests/index.test.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
 const sampleCode = fs.readFileSync(path.join(__dirname, 'sample.js'), 'utf8')
 
+delete sampleObject.func
+delete sampleObject.spreadedFunc
+
 test('compared objects are deeply equal', () => {
   const ast = parser.parse(sampleCode)
   const object = ast.program.body[0].expression.right
-  expect(equal(sampleObject, toJs(object))).toBe(true);
+  expect(toJs(object)).toEqual(sampleObject)
```
Comment (dword-design) [inline tests/index.test.js:16]: toEqual outputs diffs

**Comment by dword-design** [comment]
@hypervillain Any chance to have this merged?

**Comment by hypervillain** [comment]
Thanks a lot @dword-design, I much appreciate it!
I released the [0.0.5](https://www.npmjs.com/package/ast-to-literal/v/0.0.5) ✌️

**Comment by dword-design** [comment]
@hypervillain Awesome, thanks for merging 🙌

**Pull request merged**